app/routes/docs.tsx

```diff
@@ -13,13 +13,16 @@ import parseRemixContext from "~/data/parseRemixContext.server";
 import getMysql from "~/data/mysql.server";
 import { apps } from "data/schema";
 import { ne } from "drizzle-orm/expressions";
+import path from "path";
```

Comment:
`path` is only available in `node`, let's remove from here

Reply:
Would you mind clarifying this for me?  Are you saying that our server that hosts the docs doesn't have `node`?

Reply:
the `path` module is only available in the `nodejs` environment. Meaning, it won't be available in the browser and this is a UI component

Reply:
Ah.  Thanks!